tensorflow/python/data/experimental/kernel_tests/override_threadpool_test.py

```diff
@@ -70,17 +69,15 @@ def get_thread_id(_):
       # perform work.
       self.assertLessEqual(len(thread_ids), num_threads)
 
-  @parameterized.named_parameters(
-      ("1", 1, None),
-      ("2", 2, None),
-      ("3", 4, None),
-      ("4", 8, None),
-      ("5", 16, None),
-      ("6", 4, -1),
-      ("7", 4, 0),
-      ("8", 4, 1),
-      ("9", 4, 4),
-  )
+  @combinations.generate(
+      combinations.times(
+          test_base.default_test_combinations(),
+          combinations.combine(
+              num_threads=[1, 2, 4, 8, 16], max_intra_op_parallelism=[None]) +
+          combinations.combine(
+              num_threads=[4], max_intra_op_parallelism=[0, 1, 4]) +
```

Comment:
did you forget test case with -1 for the `max_intra_op_parallelism` value?

Reply:
The test case with `max_intra_op_parallelism=-1` is still kept. To avoid the function name conflicts, I made a little change to the original test cases. Change {`num_threads=4` & `max_intra_op_parallelism=-1`} to {`num_threads=5` & `max_intra_op_parallelism=-1`} in the last combination. Please see the detailed explanation [above](https://github.com/tensorflow/tensorflow/pull/34663#discussion_r352413125).